package.json

```diff
@@ -47,7 +47,7 @@
   },
   "jam": {
     "dependencies": {
```

Comment:
This should have really been in [`peerDependencies`](http://blog.nodejs.org/2013/02/07/peer-dependencies/) originally afaict but I am no expert. The same problem exists in Migrate.